app/javascript/mastodon/performance.js

```diff
@@ -1,7 +1,6 @@
 //
 // Tools for performance debugging, only enabled in development mode.
 // Open up Chrome Dev Tools, then Timeline, then User Timing to see output.
-// Also see config/webpack/loaders/mark.js for the webpack loader marks.
 
 import * as marky from 'marky';
```

Comment:
As we are removing the `mark-loader` that was in the webpack config, does `marky` still does something? Should it be removed?